storey/drivers.py

```diff
@@ -518,3 +518,137 @@ async def _get_item(self, container, table_path, key, attributes):
 
         return await self._v3io_client.kv.get(container, table_path, str(key), attribute_names=attributes,
                                               raise_for_status=v3io.aio.dataplane.RaiseForStatus.never)
+
+
+class SQLDriver(Driver):
+    """
+    Database connection to Sql data basw.
+    :param db_path: url string connection to sql database.
+    :param primary_key: the primary key of the table.
+    """
+
+    def __init__(
+        self,
+        primary_key: str,
+        db_path: str,
+        aggregation_attribute_prefix: str = "aggr_",
+        aggregation_time_attribute_prefix: str = "_",
+    ):
+        self._db_path = db_path
+        self._sql_connection = None
+        self._primary_key = primary_key
+        self._mtime_name = "$_mtime_"
+        self._storey_key = "storey_key"
+
+        self._aggregation_attribute_prefix = aggregation_attribute_prefix
+        self._aggregation_time_attribute_prefix = aggregation_time_attribute_prefix
+
+    def _lazy_init(self):
+        import sqlalchemy as db
+
+        if not self._sql_connection:
+            self._engine = db.create_engine(self._db_path)
+            self._sql_connection = self._engine.connect()
+
+    def table(self, table_path):
+        import sqlalchemy as db
+
+        metadata = db.MetaData()
+        return db.Table(
+            table_path[3:].split("/")[1],
```

Comment:
What's up with the `[3:]`? In a debugger I see that the table path is `'///test.db/JbNLxlQwvC'`. How did we end up with 3 slashes? This looks error prone.